test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -451,5 +477,92 @@ private static void DeleteDirectory(string path)
 
             }
         }
+
+        //temporary added methods for processing deps.json files for patching
+        private void TempPatching(string patchDir)
+        {
+            var prefix = patchDir;
```

Comment:
Remove this line.  There's no need to capture `patchDir` in a local if you're just reading it.  Just used `patchDir`.